cinema/models.py

```diff
@@ -50,8 +50,8 @@ class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
-    genres = models.ManyToManyField(Genre)
-    actors = models.ManyToManyField(Actor)
+    genres = models.ManyToManyField(Genre, blank=True)
+    actors = models.ManyToManyField(Actor, blank=True)
```

Comment:
The `ManyToManyField` for `genres` and `actors` in the `Movie` model is correctly set to allow blank entries, which aligns with the migration file. Ensure that the related models (`Genre` and `Actor`) are correctly defined and imported.